integration/network/ipvlan/ipvlan_test.go

```diff
@@ -112,22 +112,22 @@ func TestDockerNetworkIpvlan(t *testing.T) {
 	}
 }
 
-func testIpvlanSubinterface(t *testing.T, ctx context.Context, client dclient.APIClient) {
+func testIpvlanSubinterface(t *testing.T, ctx context.Context, apiClient dclient.APIClient) {
 	master := "di-dummy0"
 	n.CreateMasterDummy(ctx, t, master)
 	defer n.DeleteInterface(ctx, t, master)
 
 	netName := "di-subinterface"
-	net.CreateNoError(ctx, t, client, netName,
+	net.CreateNoError(ctx, t, apiClient, netName,
 		net.WithIPvlan("di-dummy0.60", ""),
 	)
-	assert.Check(t, n.IsNetworkAvailable(ctx, client, netName))
+	assert.Check(t, n.IsNetworkAvailable(ctx, apiClient, netName))
 
 	// delete the network while preserving the parent link
-	err := client.NetworkRemove(ctx, netName)
+	_, err := apiClient.NetworkRemove(ctx, netName, dclient.NetworkRemoveOptions{})
```

Comment:
```suggestion
	_, err := apiClient.NetworkRemove(ctx, netName, client.NetworkRemoveOptions{})
```


Or are we importing it twice? (we need to check if our linters catch that nowadays - we had some duplicate imports at times)